### DISCUSSION TITLE
32-bit Version

### DISCUSSION THREAD

**Discussion by strjh** [FormerLurker/ArcWelderLib#42] [Q&A] [answered]
Hi,
is there any chance to publish a 32-bit version of ArcWelder.exe? For some reason I can't build one myself with VS 2019 on 32-bit Win10.
Would love to try this with Prusaslicer.
Cheers :)

**Comment by FormerLurker** [comment, marked as answer]
I added one, but I think it requires the C++ runtimes (you probably have those), and I don't have a 32 bit OS to test it on.  Let me know if it works for you.  You can find it [in the latest release](https://github.com/FormerLurker/ArcWelderLib/releases).

**Reply by strjh** [reply]
Just tried it, works like a charm! Thank you so much, keep up the good work!
